src/BuildIt.Synchronization/Samples/BuildItSynchronizationSample.Core/BuildItSynchronizationSample.Core.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+    <PackageTargetFallback>portable-net45+win8+wpa81+wp8</PackageTargetFallback>
```

Comment:
Can we get rid of this value yet? I think the dependencies are all .net standard now